src/estimators.jl

```diff
@@ -83,14 +83,14 @@ end
     LGBMRegression(; [
         objective = "regression",
         boosting = "gbdt",
-        num_iterations = 10,
```

Comment:
in few cases the default parameter values differed to the official documentation defaults without any specific reason so I updated them accordingly given that all other parameters have defaults according to lightgbm documentation